static/js/services/enketo.js

```diff
@@ -18,6 +18,8 @@ angular.module('inboxServices').service('Enketo',
     'ngInject';
 
     var objUrls = [];
+    var xmlCache = {};
+    var FORM_ATTACHMENT_NAME = 'xml';
```

Comment:
It's not used anywhere else, and I pulled it out just so I could give more context about what it represents. Every time I read 'xml' I thought it was a file format instead of the attachment name. This was my attempt to make the code self documenting (at least for me).